src/main/paradox/docs/additional-info/faq.md

```diff
@@ -1,3 +1,124 @@
 # Frequently asked questions
 
-TBC.
\ No newline at end of file
+
+**What is Nexus?**
+
+Nexus is an Open Source, data and knowledge management platform designed to enable the definition of arbitrary applications domains for which there is a need to create and manage entities as well as their relations (e.g. provenance) by the FAIR principles. Indeed, Nexus enable data to be Findable, Accessible, Interoperable and Re-usable, as well as able to track data provenance and supporting data longevity in a secure and scalable manner.
+
+
+**Is Nexus free to use?**
+
+Yes, Nexus is a free, Open Source platform released under [Apache Licence 2.0](https://opensource.org/licenses/Apache-2.0)
+
+
+**How to run Nexus?**
+
+There are many ways to run Nexus: a public instance is running at the [Sandbox](https://nexus-sandbox.io/web), meanwhile if you want to run it locally you might need to install [Docker](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/docker.html) or [Minikube](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/minikube.html). You can also deploy Nexus [“on premise”](https://bluebrain.github.io/nexus/docs/getting-started/running-nexus/minikube.html), as a single instance or as a cluster.
+
+
+**How can I try Nexus without installing it? What is the difference with a regular database like PostgreSQL?**
+
```

Comment:
You can use the [cloud deployment](https://bluebrain.github.io/nexus/docs/webapps/index.html).
Although Nexus can be used as a regular database, it's flexibility and feature set are well beyond that. Just to mention some of the Nexus features:
- Allows to define different constraints to different set of data at runtime.
- Provides automatic indexing into several indexers (currently ElasticSearch and Sparql), dealing with reindexing strategies, retries and progress.
- Provides authentication
- Comes with a flexible and granular authorization mechanism.
- Guarantees resources immutability, keeping track of a history of changes.

Reply:
In cloud deployment link did you mean this https://bluebrainnexus.io//docs/getting-started/webapps.html or this https://bluebrainnexus.io//docs/getting-started/running-nexus/index.html#on-premise-cloud-deployment ?

Reply:
This one: https://bluebrainnexus.io//docs/getting-started/webapps.html